packages/adminrouter/extra/src/test-harness/modules/runner/common.py

```diff
@@ -809,6 +813,11 @@ def make_url_from_path(self, path='/exhibitor/some/path'):
 
         return base + path[1:]
 
+    @property
+    def syslog_line_buffer(self):
+        """Return line buffer where all syslog output of this object resides"""
+        return self._syslog_mock.stdout_line_buffer
```

Comment:
Thanks for fighting the Admin Router test harness! We keep it green and make @vespian proud :)